web/frontend/src/app.vue

```diff
@@ -840,15 +820,69 @@ onMounted(async () => {
     />
 
     <!-- ******* CAMERAS *******  -->
-    <CameraView
+    <v-collapse
       v-for="camera in filterResources(resources, 'rdk', 'component', 'camera')"
       :key="camera.name"
-      :camera-name="camera.name"
-      :client="client"
-      :resources="resources"
-      @selected-camera-view="t => { viewCameraFrame(camera.name, t) }"
-      @clear-interval="clearFrameInterval"
-    />
+      :title="camera.name"
+      class="camera"
+      data-parent="app"
+    >
+      <v-breadcrumbs
+        slot="title"
+        crumbs="camera"
+      />
+
+      <div class="flex flex-col gap-4 border-x border-b border-black p-4">
+        <v-switch
+          :label="camera.name"
+          :aria-label="openCameras[camera.name] ? `Hide Camera: ${camera.name}` : `View Camera: ${camera.name}`"
+          :value="openCameras[camera.name] ? 'on' : 'off'"
+          @input="setupCamera(camera.name)"
+        />
+
+        <div
+          v-if="openCameras[camera.name]"
+          class="flex flex-wrap items-end gap-2"
+        >
+          <v-select
+            v-model="refreshFrequency[camera.name]"
+            class="w-fit"
+            label="Refresh frequency"
+            aria-label="Refresh frequency"
+            :options="Object.keys(selectedMap).join(',')"
+          />
+
+          <v-button
+            v-if="refreshFrequency[camera.name] !== 'Live'"
+            icon="refresh"
+            label="Refresh"
+            @click="triggerRefresh = !triggerRefresh"
+          />
+        </div>
+
+        <CameraView
+          v-show="openCameras[camera.name]"
+          :key="camera.name"
+          :camera-name="camera.name"
+          parent-name="app"
+          :client="client"
+          :resources="resources"
+          :show-export-screenshot="true"
+          :refresh-rate="refreshFrequency[camera.name]"
+          :trigger-refresh="triggerRefresh"
+        />
+
+        <PCDView
```

Comment:
Do we have anyway of detecting if a camera can display point clouds? Would be nice to only show this when that is possible.

Reply:
This is only for the manual refresh when the camera is already in an on interval state of refreshing. This is not PCD related.